packages/bridge-ui/src/components/buttons/SelectToken.svelte

```diff
@@ -1,21 +1,28 @@
 <script lang="ts">
-  import { getProvider } from '@wagmi/core'
-  import { token } from "../../store/token";
-  import { bridgeType } from "../../store/bridge";
-  import { ETH, tokens } from "../../domain/token";
-  import type { Token } from "../../domain/token";
-  import { BridgeType } from "../../domain/bridge";
-  import { ChevronDown, PlusCircle } from "svelte-heros-v2";
-  import { errorToast, successToast } from "../../utils/toast";
-  import { ethers } from "ethers";
-  import ERC20 from "../../constants/abi/ERC20";
+  import { getProvider } from '@wagmi/core';
+  import { token } from '../../store/token';
+  import { bridgeType } from '../../store/bridge';
+  import { ETH, tokens } from '../../domain/token';
+  import type { Token } from '../../domain/token';
+  import { BridgeType } from '../../domain/bridge';
+  import { ChevronDown, PlusCircle } from 'svelte-heros-v2';
+  import { errorToast, successToast } from '../../utils/toast';
+  import { ethers } from 'ethers';
+  import ERC20_ABI from '../../constants/abi/ERC20';
   import { signer } from '../../store/signer';
   import { userTokens, tokenService } from '../../store/userToken';
   import { fromChain, toChain } from '../../store/chain';
   import Erc20 from '../icons/ERC20.svelte';
   import AddCustomErc20 from '../form/AddCustomERC20.svelte';
 
   let dropdownElement: HTMLDivElement;
+  let showAddressField = false;
+  let loading = false;
+  let customTokens = [];
+
+  userTokens.subscribe((value) => {
```

Comment:
if we are just setting customTokens to userTokens, why do we duplciate the customTokens array?

we can simple use:
```ts
{#each $userTokens as token}

{/each}
```

which will auto subscribe.

Reply:
Good point. Looks like we can simply get rid of it and use the store instead.